src/ring_theory/adjoin/basic.lean

```diff
@@ -230,3 +230,15 @@ subring.ext $ λ x, mem_adjoin_iff
 end ring
 
 end algebra
+
+open algebra subalgebra
+
+namespace alg_hom
+
+variables [comm_semiring R] [semiring A] [semiring B] [algebra R A] [algebra R B]
+
+lemma map_adjoin (φ : A →ₐ[R] B) (s : set A) :
```

Comment:
Isn't this `algebra.adjoin_image` from line 116?

Reply:
Indeed, it is! I'll close this PR now.